### PR TITLE
Polish getting started page (fix #226991)

### LINKED ISSUE #226991: Make the icons and texts in the Start container of Getting Started page vertically centered
The *Start* container in welcome page on startup looks a little disjointed because the elements are not aligned vertically.

Screenshot_20240829_010941

### PR DESCRIPTION
**Pull request by PhantomPower82**
Fix #226991: Add `vertical-align: middle` and remove `top` property.

Before:

Screenshot_20240829_010941

After:

Screenshot

### CODE REVIEW

File: src/vs/workbench/contrib/welcomeGettingStarted/browser/media/gettingStarted.css [hunk 1]
```diff
@@ -231,8 +231,6 @@
 .monaco-workbench .part.editor > .content .gettingStartedContainer .gettingStartedSlideCategories .featured-icon {
 	font-size: 20px;
 	padding-right: 8px;
-	position: relative;
```
Comment (bhavyaus) [inline src/vs/workbench/contrib/welcomeGettingStarted/browser/media/gettingStarted.css:234]: Is this change needed?

Comment (PhantomPower82) [reply src/vs/workbench/contrib/welcomeGettingStarted/browser/media/gettingStarted.css:234]: This property can be deleted or not since using `position: relative` without `top: 3px` works the same as `position: static` (default value) for the element itself (https://stackoverflow.com/a/13394093)

**Review by NriotHrreion — APPROVED** [review]
Look good.

**Review by NriotHrreion — APPROVED** [review]

**Review by bhavyaus — APPROVED** [review]

**Review by amunger — APPROVED** [review]
